cmd/fmt_test.go

```diff
@@ -153,3 +153,62 @@ p {
 		}
 	})
 }
+
+func TestFmtFailFileChanges(t *testing.T) {
+	params := fmtCommandParams{
+		fail: true,
+	}
+	var stdout bytes.Buffer
+
+	policyContent := `package test
+
+p {
+	a == 1
+	true
+	1 + 3
+}
+`
+
+	files := map[string]string{
+		"policy.rego": policyContent,
+	}
+
+	test.WithTempFS(files, func(path string) {
+		policyFile := filepath.Join(path, "policy.rego")
+		info, err := os.Stat(policyFile)
```

Comment:
Looks like we're doing this above, too, but we should either check `err` or use
```suggestion
		info, _ := os.Stat(policyFile)
```

Reply:
`err` is passed as a param on the next line

```
err = formatFile(&params, &stdout, policyFile, info, err)
```